ui/components/Navigator.js

```diff
@@ -104,8 +104,8 @@ const styles = (theme) => ({
   itemCategory: {
     backgroundColor: '#263238',
     boxShadow: '0 -1px 0 #404854 inset',
-    paddingTop: 16,
-    paddingBottom: 16,
+    paddingTop: '1.31rem',
+    paddingBottom: '1.31rem',
```

Comment:
I might be reading this out of context.

16px = 1rem

Reply:
Great. What is the other container sized to?

Reply:
The padding of right container/header is 19.2px top and bottom but because of logo size difference of right and left container and as the issue said i have to adjust the left container/sidebar container only. So after taking this sizes into consideration after increasing the left container height to 21px both height got equal. As the right container header height is near to 90px so i had to increase padding 21px. One thing i can do is to adjust logo and text sizes of right and left container to be the same and then set the same padding but it will be nearly the same thing i think.

Reply:
Great! @Aryakoste  Thanks! Looking forward for more PR's from you, I will be merging this one !